microprofile/7.1/25.0.0.7-beta-cl250620250508-1902-TCKResults.adoc

```diff
@@ -0,0 +1,50 @@
+:page-layout: certification
+= MicroProfile 7.1 TCK Results Summary
+
+As required by the https://www.eclipse.org/legal/tck.php[Eclipse Foundation Technology Compatibility Kit License], following is a summary of the MicroProfile TCK results for MicroProfile 7.1.
+
+== Open Liberty 25.0.0.7-beta-cl250620250508-1902 MicroProfile 7.1 Certification Summary
+
+MicroProfile 7.1 consists of 7 MicroProfile Specifications and it depends on the Jakarta EE Core Profile 10.0 specfication.
+
+* MicroProfile Config 3.1
+
+* MicroProfile Fault Tolerance 4.1
+
+* MicroProfile Health 4.0
+
+* MicroProfile JWT Authentication 2.1
+
+* MicroProfile OpenAPI 4.1
+
+* MicroProfile Rest Client 4.0
+
+* MicroProfile Telemetry 2.1
+
+* Jakarta EE Core Profile 10.0
+
+=== Product Name, Version and download URL (if applicable):
+
+https://public.dhe.ibm.com/ibmdl/export/pub/software/openliberty/runtime/tck/2025-05-08_1902/openliberty-25.0.0.7-beta-cl250620250508-1902.zip[Open Liberty 25.0.0.7-beta-cl250620250508-1902]
+
+In order to certify against MicroProfile 7.1, an implementation must pass the TCK tests for the seven included MicroProfile component specifications, plus the Jakarta EE Core Profile 10.0 TCK.
+
```

Comment:
Is Java 11 also needed?

Reply:
No. Tbh I'm not sure we even need more than one Java version since this is for demonstrating a compatible implementation for the spec release.